src/model/generated/vault.model.ts

```diff
@@ -19,9 +19,24 @@ export class Vault {
     @Column_("jsonb", {transformer: {to: obj => obj.toJSON(), from: obj => obj == null ? undefined : fromJsonCurrency(obj)}, nullable: false})
     collateralToken!: Currency
 
+    @Column_("numeric", {transformer: marshal.bigintTransformer, nullable: false})
+    collateralAmount!: bigint
+
     @Column_("jsonb", {transformer: {to: obj => obj.toJSON(), from: obj => obj == null ? undefined : fromJsonCurrency(obj)}, nullable: false})
     wrappedToken!: Currency
 
+    @Column_("numeric", {transformer: marshal.bigintTransformer, nullable: true})
+    pendingBtcAmount!: bigint | undefined | null
```

Comment:
Please rename to `pendingWrappedAmount` - while `wrapped` is understood to be BTC, we want to keep the nomenclature consistent. Just in case, some day, we might wrap other tokens.

Also, why do we add this field? I can't see it being set anywhere. Same for several other fields.

Reply:
I was just planning on create a pr for the amountCollateral functionality and have started working on the logic for those fields in another branch. I will convert this to a draft pr and just work on the functionality for all the fields in this branch!

Reply:
Also I renamed!